src/vl-http-400-message.js

```diff
@@ -0,0 +1,14 @@
+import {define} from '/node_modules/vl-ui-core/dist/vl-core.js';
+import {VlHttpErrorMessage} from '/src/vl-http-error-message.js';
+
+define('vl-http-400-message', class extends VlHttpErrorMessage {
+  constructor() {
+    super({
+      title: 'Oeps, er is iets misgegaan',
+      image: 'https://cdn.milieuinfo.be/http-error-message-assets/LATEST/img/unexpected-error.svg',
+      imageAlt: 'Verkeerd verzoek',
+      text: `<p>Het verzoek dat u heeft gedaan, is misgegaan door foute syntax.</p>`,
+      actions: `<a is="vl-link-button" href="/">Terug naar de startpagina</a>`,
```

Comment:
ik kan het niet direct uit de code afleiden, maar gaat het mogelijk zijn om de href en de tekst binnen deze link aan te passen? Dit is iets dat we sowieso al nodig hebben voor o.a. in IBO. Waarbij we de gebruiker bij een fout vanuit het /inspraak-beheer willen terug redirecten naar /inspraak-beheer, met als tekst "Terug naar het beheer".